rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -545,9 +563,19 @@ def decode_transaction_hashes(
                 ignore_cache=ignore_cache,
             )
             events.extend(new_events)
-            if new_refresh_balances is True:
+            if new_refresh_balances is True and refresh_balances is False:
```

Comment:
why this change?

Reply:
We are re-assigning the variable always. Made it just to avoid it

Reply:
makes no difference. The extra conditional check is more expensive than an assignment. Undo this.